hooks/pre-commit.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./git-format-staged --formatter "swiftformat --config .swiftformat stdin" "../../StreamChatClient_v3/*.swift"
```

Comment:
Instead of `./git-format-staged`, you should use `../../hooks/git-format-staged` (after you remove unnecessary symlink)